### PR TITLE
Improve docstrings for EELSSpectrum and Signal1D

### PR DESCRIPTION
**Pull request by magnunor**
Tries to improve the docstrings so they pass the tests, however the GUI ones still fail due to no way of currently running those in the tests.

### CODE REVIEW

File: hyperspy/_signals/signal1d.py [hunk 1]
```diff
@@ -98,9 +98,10 @@ def find_peaks_ohaver(y, x=None, slope_thresh=0., amp_thresh=None,
         contains position, height, and width of each peak
     Examples
     --------
-    >>> x = arange(0,50,0.01)
-    >>> y = cos(x)
-    >>> one_dim_findpeaks(y, x,0,0)
+    >>> import numpy as np
```
Comment (francisco-dlp) [inline hyperspy/_signals/signal1d.py]: There is no need to import numpy, the fixture does it and the User Guide explains that all the examples assume that plt, hs and np are in the namespace.

File: hyperspy/_signals/signal1d.py [hunk 2]
```diff
@@ -98,9 +98,10 @@ def find_peaks_ohaver(y, x=None, slope_thresh=0., amp_thresh=None,
...
+    >>> x = np.arange(0,50,0.01)
+    >>> y = np.cos(x)
+    >>> one_dim_findpeaks(y, x, 0, 0)
```
Comment (francisco-dlp) [inline hyperspy/_signals/signal1d.py]: The function is now called ``find_peaks_ohaver``

**Comment by francisco-dlp** [comment]
Appending ``#doctest: +SKIP`` to the line that calls the GUI code should help with the GUI commands.

**Comment by magnunor** [comment]
Fixed!

**Pull request merged**